### PR TITLE
MobileNet SSD sample

### PR DESCRIPTION
**Pull request by arrybn**
### This pullrequest changes
Added C++ and python samples for MobileNet-based SSD model

### CODE REVIEW

File: samples/dnn/ssd_mobilenet_object_detection.cpp [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+        {
+            cout << "Couldn't find camera" << endl;
+            return -1;
+        }
+    }
+
+    Size inVideoSize = Size((int) cap.get(CV_CAP_PROP_FRAME_WIDTH),    //Acquire input size
+                            (int) cap.get(CV_CAP_PROP_FRAME_HEIGHT));
+
+    float WHRatio = inWidth / (float)inHeight;
```
Comment (terfendail) [inline samples/dnn/ssd_mobilenet_object_detection.cpp]: Wouldn't it be better to define it near other consts?

**Comment by vpisarev** [comment]
:+1: @alalek, this PR needs to be merged in before 3.3

File: modules/dnn/src/layers/detection_output_layer.cpp [hunk 2]
```diff
@@ -234,7 +234,8 @@ class DetectionOutputLayerImpl : public DetectionOutputLayer
 
         if (numKept == 0)
         {
-            CV_ErrorNoReturn(Error::StsError, "Couldn't find any detections");
+//            CV_ErrorNoReturn(Error::StsError, "Couldn't find any detections");
```
Comment (alalek) [inline modules/dnn/src/layers/detection_output_layer.cpp]: Perhaps we should remove this line completely.

File: samples/dnn/mobilenet_ssd_python.py [hunk 3]
```diff
@@ -0,0 +1,87 @@
+from __future__ import print_function
+import numpy as np
+import sys
+sys.path.append('/home/arrybn/build/opencv/lib')
```
Comment (alalek) [inline samples/dnn/mobilenet_ssd_python.py]: Please remove this hard-coded path too. Use `PYTHONPATH` environment variable instead to `cv2.pyd` location.

File: samples/dnn/ssd_mobilenet_object_detection.cpp (hunk 1 above)
Comment (arrybn) [inline samples/dnn/ssd_mobilenet_object_detection.cpp]: fixed

File: modules/dnn/src/layers/detection_output_layer.cpp (hunk 2 above)
Comment (arrybn) [inline modules/dnn/src/layers/detection_output_layer.cpp]: fixed

File: samples/dnn/mobilenet_ssd_python.py (hunk 3 above)
Comment (arrybn) [inline samples/dnn/mobilenet_ssd_python.py]: fixed

**Pull request merged**